ai/src/ai/common/example.py

```diff
@@ -11,14 +11,36 @@
 import shutil
 from typing import Generic, Literal, TypeVar
 
-from pydantic import BaseModel
+from pydantic import BaseModel, field_validator
 
 
 class ExampleInput(BaseModel):
   prompt: str
   input_code: str | None = None
   line_number_target: int | None = None
 
+  @field_validator("prompt")
+  @classmethod
+  def is_required(cls, v):
```

Comment:
Probably could make this more generic since there are other fields that would need this validation, such as the id fields.